app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,5 +1,6 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Recursive&display=swap');
```

Comment:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/11084577/134787353-b4af82e4-7e84-46f3-862e-638b734c19ca.png">

At the bottom of this file, can do something like the screenshot:

`$subtitle-font: "Recursive", "Helvetica", "sans-serif";`

Can even replace `"Helvetica", "sans-serif";` with some other placeholders (not from google font but from more common fonts that users can fallback to in case google fonts doesn't work on them